build-logic/settings/settings.gradle.kts

```diff
@@ -1,5 +1,9 @@
 rootProject.name = "settings"
 
+plugins {
+  id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
```

Comment:
Why is this needed? (looked at the related repo and it seems like this gives us access to the zulu distro? followup question would be why not just use azul)